REVIEWS.md

```diff
@@ -0,0 +1,25 @@
+# Pull Request Reviews
+
+Whether you're contributing or reviewing pull requests, this document can help you!
+
+- If you're reviewing, it can help you understand what to do
+- If you're submitting a pull request, it can help you understand what folks
+are going to do, how to help them, and how to get your pull request
+merged faster
+
+## Steps
+
+We follow a lightweight list of steps in order to review 
+[pull requests in the Athens project](https://github.com/gomods/athens)
```

Comment:
should this be `https://github.com/gomods/athens/pulls`?

Reply:
yeah I think that's more helpful to link to